nodejs/src/wrapper.ts

```diff
@@ -46,11 +46,60 @@ const { NetInstrumentation } = require('@opentelemetry/instrumentation-net');
 const { PgInstrumentation } = require('@opentelemetry/instrumentation-pg');
 const { RedisInstrumentation } = require('@opentelemetry/instrumentation-redis');
 
+// AWS lambda instrumentation response hook for Server-Timing support
+function getEnvBoolean(key: string, defaultValue = true) {
+  const value = process.env[key];
+
+  if (value === undefined) {
+    return defaultValue;
+  }
+
+  if (['false', 'no', '0'].indexOf(value.trim().toLowerCase()) >= 0) {
+    return false;
+  }
+
+  return true;
+}
+
+// an educated guess - how to check if particular response is an API Gateway event?
+function isApiGatewayResponse(data:any) {
+    return (data && data.res && data.res.body && data.res.statusCode);
+}
+
+const responseHook = (
+    span:Span,
+    data:any
+  ) => {
+
+    const serverTimingEnabled = getEnvBoolean(
+      'SPLUNK_TRACE_RESPONSE_HEADER_ENABLED',
+      true
+    );
+    const spanContext = span.spanContext();
+
+    if (!isApiGatewayResponse(data) || !serverTimingEnabled || !isSpanContextValid(spanContext)) {
+        return;
+    }
+
+    if (!data.res.headers) {
```

Comment:
Should a check for `res` being undefined be here as well? Otherwise there might be a possibility of accessing `headers` on `undefined` which will result in a runtime error.